### PR TITLE
Define placeholders for text

### PR DESCRIPTION
**Pull request by jbmouret**
See #41 
- status bar
- text panel (top left)

Still a bug with the vertical position of the status bar when we resize the window when it is too wide.

### CODE REVIEW

File: src/robot_dart/robot_dart_simu.cpp [hunk 1]
```diff
@@ -407,18 +414,48 @@ namespace robot_dart {
...
+
+    void RobotDARTSimu::set_status_bar(const std::string& str) {
+        if (_status_bar)
+            _status_bar->text = str;
+    }
+    
+    std::string RobotDARTSimu::default_text_panel() const {
+        std::ostringstream out;
+        out.precision(3);
+        double rt = _scheduler.current_time();
```
Comment (costashatz) [inline src/robot_dart/robot_dart_simu.cpp:445]: `_scheduler.current_time()` does not give the real time passed. At the moment, only `_scheduler.step()` returns this.

File: src/examples/talos.cpp [hunk 2]
```diff
@@ -59,6 +59,8 @@ int main()
             Eigen::VectorXd commands = (init_positions + delta_pos) - robot->positions(dofs);
             robot->set_commands(commands, dofs);
         }
+        simu.set_text_panel(robot->model_filename() + "\n"
+            + simu.default_text_panel());
```
Comment (costashatz) [inline src/examples/talos.cpp:63]: It is a bit weird to have to re-set the text just for the simulation time to be updating. We could add a boolean (whether to display the default text or not), and it should be added automatically. What do you think?

File: src/robot_dart/robot_dart_simu.cpp (hunk 1 above)
Comment (jbmouret) [reply src/robot_dart/robot_dart_simu.cpp:445]: Should we update the scheduler, so?

File: src/examples/talos.cpp (hunk 2 above)
Comment (jbmouret) [reply src/examples/talos.cpp:63]: Why not. I kinda like the idea to "make your own" : here you can for instance add a title above the default (if you do not call set_text_panel, then you have the default). But I am fine with a Boolean if you prefer.

File: src/robot_dart/robot_dart_simu.cpp (hunk 1 above)
Comment (costashatz) [reply src/robot_dart/robot_dart_simu.cpp:445]: There's no other way to know the real-time factor. We need to call `scheduler.step()`; we could save this value though to be able to access it..

Comment (jbmouret) [reply src/robot_dart/robot_dart_simu.cpp:445]: Yes, this is what I had in mind. I will do it.
